### PR TITLE
Port metric v1 e2e tests

### PR DESCRIPTION
**Pull request by metamben**

### CODE REVIEW

File: e2e/test/scenarios/question/notebook.cy.spec.js [hunk 1]
```diff
@@ -845,6 +845,79 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
         .contains("[Revenue]");
     });
   });
+
+  it("should be possible to sort by metric (metabase#8283)", () => {
```
Comment (metamben) [inline e2e/test/scenarios/question/notebook.cy.spec.js:849]: Scavenged from the admin metrics tests.

File: e2e/test/scenarios/visualizations-tabular/reproductions/6010-metric-filter-drill.cy.spec.js [hunk 2]
```diff
@@ -16,30 +15,29 @@ describe("issue 6010", () => {
...
 
-    popover().findByText("See these Orders").click();
+    popover().findByText("See these Metrics").click();
     cy.wait("@dataset");
 
     cy.findByTestId("qb-filters-panel").within(() => {
-      cy.findByText("Total is greater than 150").should("be.visible");
       cy.findByText("Created At is Jan 1–31, 2024").should("be.visible");
     });
+    // FIXME metrics v2 -- check that the values in column Total are above 150
```
Comment (metamben) [inline e2e/test/scenarios/visualizations-tabular/reproductions/6010-metric-filter-drill.cy.spec.js:31]: I'm not sure how to do this best. Ideally, we would like to check that the filter is still in effect, but I don't think it should be listed as it's part of the metric which is still the source.

File: frontend/src/metabase/parameters/utils/mapping-options.unit.spec.js [hunk 3]
```diff
@@ -75,9 +75,7 @@ describe("parameters/utils/mapping-options", () => {
         });
       });
 
-      // FIXME metrics v2
-      // eslint-disable-next-line jest/no-disabled-tests
-      it.skip("should return fields from the model question's virtual card table, as though it is already nested", () => {
+      it("should return fields from the model question's virtual card table, as though it is already nested", () => {
```
Comment (metamben) [inline frontend/src/metabase/parameters/utils/mapping-options.unit.spec.js:78]: I can't see how this is related to metrics, so I think it should pass.

File: frontend/src/metabase/query_builder/containers/QueryBuilder.unsaved-changes-warning.unit.spec.tsx [hunk 4]
```diff
@@ -70,9 +70,7 @@ describe("QueryBuilder - unsaved changes warning", () => {
       expect(screen.getByTestId("leave-confirmation")).toBeInTheDocument();
     });
 
-    // FIXME metrics v2
-    // eslint-disable-next-line jest/no-disabled-tests
-    it.skip("does not show custom warning modal when saving new model", async () => {
+    it("does not show custom warning modal when saving new model", async () => {
```
Comment (metamben) [inline frontend/src/metabase/query_builder/containers/QueryBuilder.unsaved-changes-warning.unit.spec.tsx:73]: I can't see how this is related to metrics, so I think it should pass.

**Review by ranquild — APPROVED** [review]

**Pull request merged**